src/proc/Vigad.ts

```diff
@@ -101,6 +105,17 @@ export class Vigad {
         return this.captureAreas;
     }
 
+    /**
+     * Gives a notficiation after deletion of capture area
+     * @param captureAreaId
+     */
+    public deleteCaptureAreaNotification(id: number): void{
+        useNotificationSystem().createWarningNotification({
+            title: 'Capture Area Deleted',
+            message: 'The Capture Area ' + id + ' got deleted '
+        });
+    }
+
```

Comment:
Not quite the fan of this being implemented in proc/ as this is frontend... I'd rather create a wrapper function in CaptureAreaSearchValue.vue that does the notification stuff and then calls vigad.deleteCaptureArea (instead of calling vigad.deleteCaptureArea directly as click event)